### DISCUSSION TITLE
Is HEVC supported

### DISCUSSION THREAD

**Discussion by tonywingmah** [OvenMediaLabs/OvenMediaEngine#1226] [Help] [answered]
We have 50 SRT HEVC encoders can we set SRT to OME?

**Comment by getroot** [comment, marked as answer]
OME can receive HEVC from SRT. However, since LLHLS Publisher does not support HEVC yet, you need to set encoding to AVC. HEVC on LLHLS will be supported soon.

**Comment by getroot** [comment]
HEVC is now supported on SRT and LLHLS. Check out the discussion below. https://github.com/AirenSoft/OvenMediaEngine/discussions/1242
